etcdctl/ctlv3/command/user_command.go

```diff
@@ -89,3 +98,15 @@ func userAddCommandFunc(cmd *cobra.Command, args []string) {
 		ExitWithError(ExitError, err)
 	}
 }
+
+// userDeleteCommandFunc executes the "user delete" command.
+func userDeleteCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		ExitWithError(ExitBadArgs, fmt.Errorf("user delete command requires user name as its argument."))
+	}
+
+	_, err := mustClientFromCmd(cmd).Auth.UserDelete(context.TODO(), args[0])
+	if err != nil {
+		ExitWithError(ExitError, err)
```

Comment:
etcdctl v2 doesn't print message in a case of successful deletion. So I followed the manner. Should I print something in v3? I don't have a strong opinion about this.